### DISCUSSION TITLE
Every time i open zsh i have this error "omz_history:fc:14: no such event: 1"

### DISCUSSION THREAD

**Discussion by lokize** [ohmyzsh/ohmyzsh#11113] [Help] [answered]
Hi guys, after installing zsh every time i open terminal this error omz_history:fc:14: no such event: 1 appears, any way to fix that?

**Comment by lokize** [comment, marked as answer]
ok, i fixed this problem with this:

first i removed line in zshrc:

`source /root/.bashrc`

After:

Change back to bash:

`source .bashrc`

Next:

`echo $PATH`

Copy this:

`/root/youruser/....`

back to the zsh:

source .zsh

open .zshrc:

and paste:

`export PATH=/root/youruser/....`

restart terminal

Ps: in some cases like mine this path is /home not /root...

**Comment by DonRichards** [comment]
I don't know if this helps for future issues but I just ran into this and I found the adding a couple values to my .zshrc file and removing a directory history file did it for me. 

```.zshrc
# Add history-substring-search plugin (this was essential)
plugins=(history-substring-search)

# History configuration - after Oh My Zsh is sourced (possibly optional)
HISTFILE=~/.zsh_history
HISTSIZE=50000
SAVEHIST=50000

# History options (possibly optional)
setopt EXTENDED_HISTORY       # record timestamp of command in HISTFILE
setopt HIST_EXPIRE_DUPS_FIRST # delete duplicates first when HISTFILE size exceeds HISTSIZE
setopt HIST_IGNORE_DUPS       # ignore duplicated commands history list
setopt HIST_IGNORE_SPACE      # ignore commands that start with space
setopt HIST_VERIFY            # show command with history expansion to user before running it
setopt SHARE_HISTORY          # share command history data
setopt APPEND_HISTORY         # append to history file immediately

# Initialize history (this was essential)
fc -p "$HISTFILE"
```

Then I ran `rm -rf ~/.directory_history` and then reopened my terminal.
